frontend/src/APIClients/mutations/TestMutations.ts

```diff
@@ -1,11 +1,49 @@
 import { gql } from "@apollo/client";
 
-const DUPLICATE_TEST = gql`
+export const PUBLISH_TEST = gql`
+  mutation PublishTest($id: ID!) {
+    publishTest(id: $id) {
+      id
```

Comment:
for now, we can just return the id itself since we won't be using the rest of the information